### PR TITLE
Adds indicator documentation to API

### LINKED ISSUE #269: Add documentation pages for indicators endpoints

### PR DESCRIPTION
**Pull request by BobTorgerson**
This PR adds the indicator documentation to the API. 

Closes #269

### CODE REVIEW

File: templates/documentation/indicators.html [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+      <td>Era-based min-mean-max for all climate indicator variables at a given latitude and longitude.</td>
+      <td>
+        <a href="/indicators/base/point/61.5/-147">/indicators/base/point/61.5/-147</a>
+      </td>
+    </tr>
+    <td>Era-based min-mean-max for all climate indicator variables for a specific
+      <a href="/places/all">area of interest polygon ID</a> and aggregate by mean.
+    </td>
+    <td>
+      <a href="/indicators/base/area/61.5/-147">/indicators/base/area/61.5/-147</a>
```
Comment (charparr) [inline templates/documentation/indicators.html:41]:
```suggestion
      <a href="/indicators/base/area/61.5/-147">/indicators/base/area/GMU7</a>
```
map area example to an area, not a lat-lon

File: templates/documentation/indicators.html [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+
+<pre>
+{
+  &lt;variable&gt;: {
+    &lt;era&gt;: {
+      &lt;model&gt;: {
+        &lt;scenario&gt;: {
+          "max": &lt;monthly sum of daily evapotranspiration, mm&gt;,
+          "mean": &lt;monthly sum of daily glacier ice melt, mm&gt;,
+          "min": &lt;monthly maximum of daily ice water equivalent, mm&gt;
```
Comment (charparr) [inline templates/documentation/indicators.html:104]: These are the hydrology outputs, not the indicators :)

**Review by charparr — CHANGES_REQUESTED** [review]
See my two comments for changes, and I'd also suggest that we copy-paste the list of variables from the CSV download and add them to the documentation. What indicators are here, and what are the definitions? If nothing else it'd probably help us in the future when we forget exactly what is in this endpoint!

File: templates/documentation/indicators.html [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+          ...
+        },
+        ...
+      },
+    ...
+    },
+    ...
+  },
+  ...
+}
```
Comment (kyleredilla) [inline templates/documentation/indicators.html:91]:
```suggestion
{
  "cd": {
    "historical": {
      "Daymet": {
        "historical": {
          "max": -30.3,
          "mean": -37.6,
          "min": -45.2
        },
      },
    },
    longterm": {
      "MRI-CGCM3": {
        "rcp45": {
          "max": -21.9,
          "mean": -34.0,
          "min": -42.2
        },
        "rcp85": {
          "max": -27.2,
          "mean": -33.4,
          "min": -40.8
        }
      },
      ...
    },
    ...
  },
  ...
}
```

File: templates/documentation/indicators.html
Comment (kyleredilla) [inline templates/documentation/indicators.html:1]: Agree with Charlie's comment on adding indicators, here they are from the supplemental info in the metadata record:

hd: “Hot day” threshold -- the highest observed daily maximum 2 m air temperature such that there are 5 other observations equal to or greater than this value.

cd: “Cold day” threshold -- the lowest observed daily minimum 2 m air temperature such that there are 5 other observations equal to or less than this value.

rx1day: Maximum 1-day precipitation

su: "Summer Days" –- Annual number of days with maximum 2 m air temperature above 25 C

dw: "Deep Winter days" –- Annual number of days with minimum 2 m air temperature below -30 C

wsdi: Warm Spell Duration Index -- Annual count of occurrences of at least 5 consecutive days with daily mean 2 m air temperature above 90th percentile of historical values for the date

cdsi: Cold Spell Duration Index -- Same as WDSI, but for daily mean 2 m air temperature below 10th percentile

rx5day: Maximum 5-day precipitation

r10mm: Number of days with precipitation > 10 mm

cwd: Consecutive wet days –- number of the most consecutive days with precipitation > 1 mm

cdd: Consecutive dry days –- number of the most consecutive days with precipitation < 1 mm


May want to format better with degree symbols etc.

**Review by charparr — APPROVED** [review]

**Comment by charparr** [comment]
Doc page looks great after most recent changes and is a good match for the GN record, thanks Bob!

**Pull request merged**